README.md

```diff
@@ -102,7 +102,34 @@ app.post("/api/testbox/trial", async (req, res) => {
     // to fulfill the request. Whenever possible, respond to TestBox synchronously
     // using a 201 HTTP code. 200 HTTP codes will be ignored, as we will assume
     // that you are creating the trial asynchronously.
-    trialRequest.express.fulfill(trial, res);
+    trialRequest.express.fulfill(testboxTrial, res);
+});
+```
+
+## Setup use cases
+
+```typescript
+import { TestBoxUseCaseRequest, TestBoxTrial } from "@testboxlab/node-sdk";
+
+const app = express();
+
+// You can use the use-case call to return a URL for 
+// the requested use-cases
+app.post("/api/testbox/use-cases", async (req, res) => {
+    const useCaseRequest = new TestBoxUseCaseRequest(req.body);
+    const tokenVerified = await useCaseRequest.verifyToken(req.headers["authorization"]);
+    if (!tokenVerified) {
+        // The token verification failed, meaning someone is trying to pretend to be
+        // TestBox! Do not process their request.
+        return res.status(401).send();
+    }
+
+    // You may now safely retrieve a URL for the requested use case
+    const useCaseType = useCaseRequest.types[0]
+    url = myUseCaseUrlRetrieveFunction(useCaseRequest)
+    // Once we have finished build the URL for the use case, we need
+    // to fulfill the request. 
+    useCaseRequest.express.fulfill(useCaseType, url, res);
```

Comment:
some thoughts... instead of having to call trialRequest.express.fullfill, we should initialize this in such a way that we initialize the node SDK with an express provider. So that no matter the node framework, interaction with the testbox sdk is exactly the same

Reply:
so you could just call trialRequest.fulfill

Reply:
Hmmm, that would be nice.
Tho if we changed this it would also be nice to follow the same standard for trial.
So is this something we want to do now or as a follow up?

Reply:
I guess go ahead and do as a follow up so we have smaller PRs